Assets/LeapMotion/Scripts/HandController.cs

```diff
@@ -33,6 +33,12 @@ public class HandController : MonoBehaviour {
   protected const float GIZMO_SCALE = 5.0f;
   /** Conversion factor for millimeters to meters. */
   protected const float MM_TO_M = 0.001f;
+  /** Conversion factor for nanoseconds to seconds. */
+  protected const float NS_TO_S = 0.000001f;
```

Comment:
For clarity, it should be possible to use scientific notation. For example NS_TO_S = 1e-6f,